web/assets/js/views/PhysicalEvaluation-0.3-ehr.js

```diff
@@ -738,7 +738,7 @@ PMI.views['PhysicalEvaluation-0.3-ehr'] = Backbone.View.extend({
         this.render();
         var today = new Date();
         var sixMonthsAgo = (new Date()).setMonth(today.getMonth() - 6);
-        $('.ehr-date').pmiDateTimePicker({
+        $('input.ehr-date').not(':disabled').pmiDateTimePicker({
```

Comment:
I don't know if this applies to this field, but there are situations where some fields on the form may be hidden or disabled due to branching logic. If the form was initially loaded with a date field disabled, and then other changes caused the field to be enabled, the date picker wouldn't be applied. Would it make more sense to check `if (!this.finalized)`?